src/pages/schedule-interview/components/MobileCandidateConfirmation/MobileCandidateConfirmation.tsx

```diff
@@ -0,0 +1,65 @@
+import { CalendarOutlined } from "@ant-design/icons"
+import { PHIcon } from "@ph/ui"
+import React from "react"
+
+import { cn } from "@/utils"
+import { formatDateToLongString, formatDateToTimeSlot } from "@/utils/dateTime"
+
+import { LIInterviewMode, LIInterviewModeLabel } from "../MobileCandidateSelectInterviewTime/constants"
+import { useCandidateConfirmation } from "./hooks/useCandidateConfirmation"
+import styles from "./styles.module.css"
+
+export const MobileCandidateConfirmation = () => {
+  const { interviewData } = useCandidateConfirmation()
+  const { jobRequisitionName, interviewMode, interviewAddress, timezone, interviewTime, interviewLink } = interviewData
+  const interviewDate = formatDateToLongString(new Date(interviewTime.start), timezone)
+  const interviewTimeSlot = formatDateToTimeSlot(interviewTime, timezone)
+
+  return (
+    <article
+      className={cn([
+        "candidateConfirmationWrapper h-screen overflow-y-auto bg-gray-100 p-5",
```

Comment:
why do we still have `candidateConfirmationWrapper` here?
```suggestion
        "h-screen overflow-y-auto bg-gray-100 p-5",
```